catalog/app/containers/Auth/saga.js

```diff
@@ -150,6 +152,19 @@ function* signIn(credentials) {
   }
 }
 
+function* setBrowseCookie(tokens) {
+  try {
+    yield call(apiRequest, {
+      auth: { tokens, handleInvalidToken: false },
+      url: `${cfg.s3Proxy}/browse/set_browse_cookie`,
+      method: 'POST',
+      credentials: 'include',
+    })
+  } catch (e) {
+    log.warn('Unable to set browse cookie', e)
```

Comment:
let's add `Sentry.captureException(e)` here, because this is something we should be aware of